src/MacroTools/FactionSystem/Faction.cs

```diff
@@ -518,7 +519,8 @@ public void Obliterate()
     /// </summary>
     public IEnumerable<Power> GetAllPowers()
     {
-      foreach (var power in _powers) yield return power;
+      foreach (var power in _powers.OrderBy(p => p.Name)) 
```

Comment:
I actually don't think we want to sort this one by name. Sorting it by order of acquirement is better.